lib/widgets/info_card.dart

```diff
@@ -6,104 +6,136 @@ import '../resources/theme.dart';
-                            .copyWith(fontSize: 18),
-                        overflow: TextOverflow.fade,
-                        maxLines: 2,
-                      ),
-                    ),
-                  if (onShowMore != null)
-                    GestureDetector(
-                      onTap: onShowMore,
-                      key: showMoreButtonKey ?? const ValueKey('show_more'),
-                      child: Row(
-                        children: <Widget>[
-                          Text(
-                            S.current.actionShowMore,
-                            style: Theme.of(context)
-                                .coloredTextTheme
-                                .subtitle2
-                                .copyWith(
-                                    color: Theme.of(context).primaryColor),
-                          ),
-                          Icon(
-                            Icons.arrow_forward_ios_outlined,
-                            color: Theme.of(context).primaryColor,
-                            size:
-                                Theme.of(context).textTheme.subtitle2.fontSize,
-                          )
-                        ],
-                      ),
-                    ),
-                ],
-              ),
-              const SizedBox(height: 10),
-              FutureBuilder(
-                  future: future,
-                  builder: (final context, final snapshot) {
-                    if (snapshot.connectionState == ConnectionState.done) {
-                      if (snapshot.hasData) {
-                        if ((snapshot.data is Map ||
-                                snapshot.data is Iterable) &&
-                            snapshot.data.isEmpty) {
-                          return noneYet(context);
-                        }
+          child: Padding(
+              padding: const EdgeInsets.all(12),
+              child: Column(
+                  children: <Widget>[
+                    cardBar,
+                    const SizedBox(height: 10),
+                    content,
+                  ]
+              )
+          )
+      )
+  );
 
-                        return builder(snapshot.data);
-                      } else {
-                        return noneYet(context);
-                      }
-                    }
-                    return const SizedBox(
-                      height: 100,
-                      child: Center(child: CircularProgressIndicator()),
-                    );
-                  }),
-            ],
-          ),
-        ),
+  Widget noneYet(
+      final BuildContext context) => SizedBox(
+    height: 100,
+    child: Center(
+      child: Text(
+        S.current.warningNoneYet,
+        style: TextStyle(color: Theme.of(context).disabledColor),
       ),
-    );
-  }
+    ),
+  );
 
-  Widget noneYet(final BuildContext context) => SizedBox(
-        height: 100,
-        child: Center(
+  Widget cardBar(
```

Comment:
Rename to cardHeader, "bar" has a bit of a different meaning (e.g. appbar, toolbar)